src-plugins/itkProcessRegistrationDiffeomorphicDemons/itkProcessRegistrationDiffeomorphicDemonsPlugin.cpp

```diff
@@ -49,7 +49,12 @@ QStringList itkProcessRegistrationDiffeomorphicDemonsPlugin::contributors() cons
 
 QString itkProcessRegistrationDiffeomorphicDemonsPlugin::description() const
 {
-    return tr("Applies the diffeomorphic demons as they can be found in itk. Converts any type of image to float before applying the change, since the diffeomorphic demons only work on float images <br/> see: <a href=\"http://www.insight-journal.org/browse/publication/154\" > http://www.insight-journal.org/browse/publication/154 </a>");
+    QString description = \
+            tr("Applies the diffeomorphic demons as they can be found in the <a href=\"https://github.com/Inria-Asclepios/RPI\" style=\"color: #cc0000\" >RPI library</a>. <br><br>\
```

Comment:
This should be "Applies the diffeomorphic demons algorithm implemented in (adresse RPI) ".
For the line below: (...) since the diffeomorphic demons algorithm only works (...)

Reply:
Correction: for the first line keep the "the" at the end before the library link. I didn't see there was the work library in the link.